projects/i18n/src/lib/pipe/translation-pipe.spec.ts

```diff
@@ -15,32 +15,6 @@ class GeneralTranslationPipe extends TranslationPipe {
     }
 }
 
-class MockChangeDetectorRef extends ChangeDetectorRef {
-    constructor() {
-        super();
-    }
-
-    markForCheck(): void {
-        throw new Error('Unimplemented');
-    }
-
-    detach(): void {
-        throw new Error('Unimplemented');
-    }
-
-    detectChanges(): void {
-        throw new Error('Unimplemented');
-    }
-
-    checkNoChanges(): void {
-        throw new Error('Unimplemented');
-    }
-
-    reattach(): void {
-        throw new Error('Unimplemented');
-    }
-}
-
 const changeRef = {
```

Comment:
Do we even need this? Can't we use the live one for angular?
https://angular.io/guide/testing#componentfixture-properties `changeDetectorRef `?